### DISCUSSION TITLE
Monorepo deployment with turbo is broken

### DISCUSSION THREAD

**Discussion by maggo** [vercel/community#1218] [Help]
### Summary

Seems like the default build command for a monorepo with turbo has changed and is applying a broken filter rule.

Now broken:

```
Running "cd ../.. && npx turbo run build --filter=react..."
21:42:55.872 | • Packages in scope:
21:42:55.872 | • Running build in 0 packages
21:42:55.872 | • Remote caching enabled
21:42:55.872 |  
21:42:55.872 | Tasks:    0 successful, 0 total
21:42:55.873 | Cached:    0 cached, 0 total
21:42:55.873 | Time:    17ms
```

Worked yesterday:

```
Running "cd ../.. && npx turbo run build --filter={./examples/react}..."
20:36:48.829 | • Packages in scope: @maggo/use-flow, @maggo/use-flow-react-example
20:36:48.829 | • Running build in 2 packages
20:36:48.830 | • Remote caching enabled
```

### Example

https://github.com/maggo/use-flow

**Comment by samcx** [comment]
@maggo Hi there, is it possible to share with us both Deployment URLs?

**Reply by maggo** [reply]
Working: https://vercel.com/muchwin/use-flow-example/Dr6WYm6jS1dGXoGW1X613hByvXEX
Failed: https://vercel.com/muchwin/use-flow-example/AT4SuyB7HqARJs2FEfg45QjgyY9X

**Reply by maggo** [reply]
Any update here? Saw that you forked the repo for testing :)

**Reply by relferreira** [reply]
I'm with the same problem

Working: https://vercel.com/scribo/scribo-dashboard/J4cb6Xg41qLYjkuV9D8Ja7uHR9Fk
Failed: https://vercel.com/scribo/scribo-dashboard/CrsAkke2nwibtGr9fnrnH2FqYvL3

**Reply by tknickman** [reply]
Hey thanks for the reports - we'll take a look into why this default command changed. Are you able to unblock by manually changing the build command back to the previously known working version in your project settings?

**Reply by maggo** [reply]
Changed my vercel build command to `npx turbo run build --filter=@maggo/use-flow-react-example` and it's working fine. Thank you for looking into it!

**Reply by peterferguson** [reply]
hey having a similar problem where the same commit deploys and then the next day it doesn't. Also using turbo repo but the fix above didn't work for me. 

Seems that the repo isn't being cloned correctly:

![image](https://user-images.githubusercontent.com/7002211/212431746-23af72e3-c9ce-4722-aae3-16c627059c99.png)

**Reply by relferreira** [reply]
Sorry, I forgot to reply @tknickman, but changing the build command to 
```
cd ../.. && npx turbo run build --filter={./apps/render}...
```
Fixed the problem.
Thank you
